lib/stores/FileStore.ts

```diff
@@ -180,14 +166,14 @@ class FileStore extends DataStore {
           return reject(ERRORS.FILE_NOT_FOUND)
         }
 
-        config.size = stats.size
-        return resolve(config)
+        file.size = stats.size
```

Comment:
Is it not problematic to modify the returned object from configstore.get? Considering the MemoryConfigStore, which only stores the objects in a big map, one request can modify the entry without calling configstore.set. Maybe we should do a deep clone in configstore.get?